domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewScheduleDateCalculationService.kt

```diff
@@ -132,7 +131,7 @@ class ReviewScheduleDateCalculationService(
     }
 
   private fun reviewScheduleCalculationRuleBasedOnTimeLeftToServe(releaseDate: LocalDate): ReviewScheduleCalculationRule {
-    val timeLeftToServe = MonthsAndDaysLeftToServe.until(releaseDate)
+    val timeLeftToServe = from(baseScheduleDate()).until(releaseDate)
```

Comment:
This was the core fix - previously we were correctly calculating time left to serve ... but it was always based from "today"
The business rules for these calculations in respect of working out time left to serve to then work out their Review Schedule say that they are calculated "as if the induction was conducted on 1st April" (when we are working out Review Schedules for prisoners before the go live)

The `baseScheduleDate()` function already existed , and returns the later of "today" or the injected earliest date (April 1st)